tools/src/code-review/index.ts

```diff
@@ -95,7 +105,11 @@ export async function reviewPullRequestAsync(prNumber: number) {
 
   // Submit a review if there is any review comment (usually suggestion).
   if (reviewComments.length > 0) {
-    await submitReviewWithCommentsAsync(pr.number, reviewComments);
+    // As described on GitHub's API docs (https://docs.github.com/en/rest/pulls/reviews#create-a-review-for-a-pull-request),
+    // submitting a review triggers notifications and thus is a subject for rate limiting.
+    // Even though this sends just one request, we've got rate limited once when we sent included many comments.
+    // As an attempt to prevent that, we limit the number of comments.
+    await submitReviewWithCommentsAsync(pr.number, reviewComments.splice(0, COMMENTS_LIMIT));
```

Comment:
To not exceed the number of comments we could use the warnings mechanism:
![image](https://user-images.githubusercontent.com/9578601/183600264-0e40b300-dac9-4487-81cc-7b00ba32eacb.png)

Reply:
Yeah, I was thinking about using this feature, but since I'm planning to add suggestions to fixable issues, it wouldn't help too much (it's not supported there yet)